V0.3

anitagraser opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

anitagraser committed:
Update meta.yaml

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

anitagraser commented:
@conda-forge-admin please rerender

anitagraser committed:
Pin bokeh<2.0

Like in setup.py https://github.com/anitagraser/movingpandas/commit/388a2cfe0251914c9ea06146b938a6ab78b65769

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * ``requirements: run: bokeh<2.0`` must contain a space between the name and the pin, i.e. ``bokeh <2.0``

anitagraser committed:
Fix syntax

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.
